### ISSUE TITLE
Can not run EXE project

### ISSUE THREAD

**Issue by volehuy1998** [microsoft/Windows-driver-samples#659]
VS2019 Windows Filtering Platform **EXE project**
I turned on testsigning and debug. But it didn't work.
![WFP EXE error](https://user-images.githubusercontent.com/36606761/130548409-cfdf59f2-3b5b-44bc-bb4d-200aed6372c0.PNG)

**Comment by volehuy1998** [comment]
![WFP EXE error](https://user-images.githubusercontent.com/36606761/130548359-b53ff3a4-03a6-4edb-80d9-d2eeac9efae9.PNG)

**Comment by tqk2811** [comment]
Me too, reason here  
![image](https://user-images.githubusercontent.com/13986296/135201318-7157118c-d1ef-4ad6-8c23-993b8c0ba73d.png)
-----------------------------
I was try turn off [Secure Boot](https://answers.microsoft.com/en-us/protect/forum/all/your-organization-used-windows-defender/ef12b7be-41f5-4b71-a73c-e4f99ba944f7) but it not work

**Comment by volehuy1998** [comment]
@tqk2811 I have solved it, just missing something about strong digital signature. Let try with me.
We will use some already tool in this folder: makecert.exe, pvk2pfx.exe and signtool.exe
`
cd "C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64\"
`
After that, run this command
```
makecert.exe -sv MyApp.pvk -n "CN=MyCompany" -len 2048 MyApp.cer -r
pvk2pfx.exe -pvk MyApp.pvk -spc MyApp.cer -pfx MyApp.pfx -po MyPassword
signtool.exe sign /f MyApp.pfx /p MyPassword "Release\WFPSampler.exe"
```
Use signtool for all all of WFPSampler solution output (driver, service, ...).

After sign all output, I've deployed and test on virtual machine (bridge static ip192.168.1.62), it worked very well. Below image that WFPSampler blocked packet come from Google DNS server (8.8.8.8)
![image](https://user-images.githubusercontent.com/36606761/135205688-5ab5d23b-d151-4156-816b-3406dd903841.png)
